examples/custom_distribution.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.            return generate_samples(genpoisson_rvs,
</pre><p>The code formatting should already be compatible with our pre-commit. In this case that'll be a line break after <code>generate_samples(</code>  followed by 4-space indentation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>